src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Logging/LogProviders.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace NewRelic.Agent.Extensions.Logging
+{
+    public enum LogProvider
+    {
+        Log4Net,
+        Serilog,
+        NLog
+    }
+
+    public static class LogProviders
+    {
+        public static readonly bool[] RegisteredLogProvider = new bool[3];
```

Comment:
Could consider doing something like `new bool[Enum.GetNames(typeof(LogProvider)).Length]` just to prevent needing to remember to update the hardcoded array length if we add another provider in the future.

Reply:
I like that!